src/unix/bsd/freebsdlike/mod.rs

```diff
@@ -1593,6 +1593,14 @@ extern "C" {
     pub fn iconv_close(cd: iconv_t) -> ::c_int;
 }
 
+extern "C" {
+    // Added in `FreeBSD` 11.0
+    // Added in `DragonFly BSD` 5.4
+    pub fn explicit_bzero(s: *mut ::c_void, len: ::size_t);
+    // ISO/IEC 9899:2011 ("ISO C11") K.3.7.4.1
+    pub fn memset_s(s: *mut ::c_void, smax: ::rsize_t, c: ::c_int, n: ::rsize_t) -> ::errno_t;
```

Comment:
macOs has also memset_s if that helps.

Reply:
Added `memset_s` to macOS and fixed the signature to use standard C types instead of the C11 annex K types that were used.